src/widgets/block.rs

```diff
@@ -1100,15 +1107,60 @@ mod tests {
         const _DEFAULT_STYLE: Style = Style::new();
         const _DEFAULT_PADDING: Padding = Padding::uniform(1);
         const _DEFAULT_BLOCK: Block = Block::new()
-            .title_style(_DEFAULT_STYLE)
+            // the following methods are no longer const because they use Into<Style>
+            // .style(_DEFAULT_STYLE)           // no longer const
+            // .border_style(_DEFAULT_STYLE)    // no longer const
+            // .title_style(_DEFAULT_STYLE)     // no longer const
             .title_alignment(Alignment::Left)
             .title_position(Position::Top)
             .borders(Borders::ALL)
-            .border_style(_DEFAULT_STYLE)
-            .style(_DEFAULT_STYLE)
             .padding(_DEFAULT_PADDING);
     }
 
+    /// This test ensures that we have some coverage on the Style::from() implementations
+    #[test]
+    fn block_style() {
```

Comment:
I think this ensures coverage for `Into` implementations. `From` are already tested in `style.rs`.

Either way I'd prefer this to stay in `style.rs`.
If we need coverage for `Into` we could do something like, instead of `from` in `style.rs`:
```rust
assert_eq!(Style::default().red(), Color::Red.into::<Style>());
```

Reply:
I could have sworn I replied to this, but I must have closed the tab without hitting send.

These tests are about testing from the perspective of the user, who will rarely need to call `style::From` and instead will be passing values into these methods. From and Into are two sides of the same coin. Testing the From impl means that the Into impl will always work, but these tests say that calling the style methods is an anticipated way to interact with the API.

We could test all the Into<Style> methods, but I think that's overkill and unnecessary.

Reply:
Okay, I see your point now. Fine by me then!